WordPress/Classes/Login/LoginWithUrlView.swift

```diff
@@ -112,10 +108,7 @@ private extension LoginWithUrlView {
 // MARK: - SwiftUI Preview
 
 #Preview {
-    LoginWithUrlView(
-        client: .init(session: .shared),
-        anchor: mockAnchor
-    ) { _ in }
+    LoginWithUrlView(presenter: nil) { _ in }
 }
 
 private let mockAnchor = ASPresentationAnchor()
```

Comment:
(nit) unused

Reply:
Good catch! Removed in https://github.com/wordpress-mobile/WordPress-iOS/pull/24444/commits/61b7f47b370148d4dec9752bc4f4ef9e2e0206c4